linera-storage/src/lib.rs

```diff
@@ -173,6 +177,54 @@ pub trait Storage: Sized {
         information: &NetworkDescription,
     ) -> Result<(), ViewError>;
 
+    /// Returns a map of the committees for the given epochs.
+    async fn committees_for(
+        &self,
+        mut epochs: BTreeSet<Epoch>,
+    ) -> Result<BTreeMap<Epoch, Committee>, ViewError> {
+        let read_committee = async |committee_hash| -> Result<Committee, ViewError> {
+            let blob_id = BlobId::new(committee_hash, BlobType::Committee);
+            let committee_blob = self
+                .read_blob(blob_id)
+                .await?
+                .ok_or_else(|| ViewError::NotFound(format!("Blob not found: {}", blob_id)))?;
+            Ok(bcs::from_bytes(committee_blob.bytes())?)
+        };
+
+        let network_description = self
+            .read_network_description()
+            .await?
+            .ok_or_else(|| ViewError::NotFound("NetworkDescription not found".to_owned()))?;
+        let admin_chain_id = network_description.admin_chain_id;
+        let epoch_creation_events = self
+            .read_events_from_index(&admin_chain_id, &StreamId::system(EPOCH_STREAM_NAME), 0)
```

Comment:
Maybe the starting index should only be the minimum of the epochs we're interested in.

And we should also allow specifying an _upper_ bound for the event index at some point. (Not urgent, though; in practice it'll usually be the most recent events that are interesting.)